content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,113 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule pods.
```

Comment:
I miss some "why" - why does the scheduler need to use the capacity information? To provision volumes on nodes (or topology segment) that actually have some free space.

Reply:
I added ".... scheduler uses that information to schedule pods onto nodes
that have access to enough storage capacity for the remaining missing
volumes. Without storage capacity tracking, it is random whether the selected
node can run the Pod and multiple scheduling retries may be needed."